scripts/globals/items/tube_of_clear_salve_ii.lua

```diff
@@ -0,0 +1,44 @@
+-----------------------------------------
+-- ID: 5838
+-- Item: tube_of_clear_salve_ii
+-- Item Effect: Instantly removes all negative status effects from pet
+-----------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    if (not target:hasPet()) then
```

Comment:
No need for these braces:
https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations